modules/salt/pages/yomi.adoc

```diff
@@ -96,135 +129,150 @@ cobbler profile add \
 * Theme: selected
 
-The 'Kernel' parameter is the one than can be used for the GRUB 'append' section.
+The [parameter]``Kernel`` parameter can be used for the GRUB ``append`` section.
 
-=== Yomi Software
 
-Here we will list the different repositories and packages that we want to install. We also have a chance to register the product using 'SUSEConnect', and install the different modules after the registering.
+Yomi Software::
 
-For this example we are going to install a very minimal openSUSE Tumbleweed distribution, using the public repositories from Internet. For real deployments you must provide a local repository.
+This form provides the different repositories and packages to install.
+You can also register the product in this form, using SUSEConnect, and install the different modules after registering.
 
-Add a new repository (click on '+' close to the 'Repositories' section)
+For this example we are going to install a very minimal openSUSE Tumbleweed distribution, using publicly available repositories.
+For production deployments, you will need to provide a local repository.
+
+Add a new repository:
 * Repository Name: repo-oss
 * Repository URL: http://download.opensuse.org/tumbleweed/repo/oss/
 
-Add three packages (click on '+' close to the 'Packages' section)
+Add these packages:
 * pattern:enhanced_base
 * glibc-locale
 * kernel-default
 
-Note that we can add patterns and products, together with packages, using the correct prefix.
+You can also add patterns and products, together with packages, by using the correct prefix.
+
+
+Yomi Services::
+
+By default Yomi is installed with the ``salt-minion`` service, but you must enable it.
 
-=== Yomi Services
+Add a new enabled service:
 
-By default Yomi is install the ``salt-minion`` service, but you must to validate the enable of it.
+* Service 1:
+** Service: salt-minion
 
-Add a new enabled service (click on '+' close to the 'Enabled' section)
 
-Set for 'Service 1':
-* Service: salt-minion
+Yomi Users::
 
-=== Yomi Users
+This form sets out the system users.
+In this example, we have a single root user.
+To provide a password, you must use the hashed version of the password, not the plain text.
+This behavior is set to be changed in future versions of Yomi.
 
-For this example system we are going to create the 'root' user, under the 'linux' password. To provide a password we need to use the hashed version of it (but this will change in future versions of Yomi)
+* User 1:
+** Username: root
+** Password Hash: $1$wYJUgpM5$RXMMeASDc035eX.NbYWFl0
 
-Set for User 1:
-* Username: root
-* Password Hash: $1$wYJUgpM5$RXMMeASDc035eX.NbYWFl0
 
 
-== Monitoring the installation
+== Monitor the Installation
 
-Optionally we can monitor the installation (that will be commanded once we apply the highstate to the new system). For this the ``salt-api`` service is required.
+You can monitor the installation as it progresses, using the ``monitor`` tool from the ``salt-api`` service.
```

Comment:
'monitor' is a tool from yomi, and for work requires the 'salt-api' service, from salt. I would phrase it like:

using the monitor too from Yomi. This tool requires the activation of the salt-api service. For more information about the ``salt-api`` service, and how to use the ``monitor`` tool that it provides, see https://github.com/openSUSE/yomi.